### DISCUSSION TITLE
Is it possible to disable the privacy badge enabling users to download the Tor browser?

### DISCUSSION THREAD

**Discussion by Earthpac** [globaleaks/globaleaks-whistleblowing-software#3380] [Q&A]
### Proposal

Hi! It seems that I can't find a way to disable "Download Tor-browser" warning from the upperhand corner of the login page. Is there a way to do that or am I simply missing something?

### Motivation and context

Especially in industrial setting there are policies that prevent installing applications that are not whitelisted. The warning message tends to confuse users that are not tech savvy and clutters the screen.

**Comment by evilaliv3** [comment]
Thank you @Earthpac

We advice to never disable this banner that is fundamental to inform users about the privacy features of the software they use and guide them to use the best options.

You could anyhow disable this banned in the Site Settings - Advanced; the option is called "privacy badge".

If you disable the privacy badge we advice to talk about the possibility of using the Tor browser in the whistleblowing policy of your corporate and to inform users to never blow the whistle from the intranet of the corporate.

**Reply by fiveGiGuy** [reply]
Hello @evilaliv3 

I reply here because I think it's the same subject. 

Although I write a custom message in this privacy warning box through advanced settings, I cant configure it for not working as a toggle button. I wish it was always visible.

Can you help me, please?

Thanks

**Reply by lakshaa** [reply]
Hello, did you manage to make it always visible? @jpdionisio
